app/main/helpers/search_helpers.py

```diff
@@ -218,3 +218,10 @@ def valid_page(page):
         except ValueError:
             return None
     return None
+
+
+def map_framework_slug_to_display_service_framework_slug(slug):
```

Comment:
This conflicts with the fix I just landed - sorry! Basically, we don't need it, cos it's covered by the map I've added into the config.

One difference is I didn't bother mapping G-Cloud 7 and 8 because we do actually have content for those (even though it wasn't used, and is as far as I could see identical the the G-Cloud 6 content).